metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -50,6 +50,7 @@ entities:
       - domains
       - container
       - deprecation
+  # Note: This is in BETA version
```

Comment:
nit: can you add this to the doc field like the case of corpuser below? does it cause build issues?